simple-post-view/ideadb_handler.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+from boto3.dynamodb.conditions import Key
+
```

Comment:
Feel free to include logging facilities here as well. Same process as `function.py` to set up and use. The more observable, the better

Reply:
The thing is I am using logger in function.py where ideadb_handler is used. So If I add logger in ideadb_handler then it will be logged twice. 
Should I still add it?

Reply:
It should be a different logger, so it will be to be reconfigured in python. Logs in `ideadb_handler` should show insight to how it works, typically separate from `function` logs. If you still believe them unnecessary, mark this as resolved and focus on the other changes.